Makefile

```diff
@@ -54,27 +54,27 @@ clone:
 build_play-cpp-sdk: clone
 ifeq ($(shell uname -m), x86_64)
  ifeq ($(UNAME), Darwin)		
-  ifeq ($(USE_ARM64),true)
+	cargo install cargo-lipo
 	rustup target add aarch64-apple-darwin
-	MACOSX_DEPLOYMENT_TARGET=10.15 CXX=$(CXX) CXXFLAGS=$(CXXFLAGS) cargo build --package play-cpp-sdk --release --target aarch64-apple-darwin
-  else
 	rustup target add x86_64-apple-darwin
-	MACOSX_DEPLOYMENT_TARGET=10.15 CXX=$(CXX) CXXFLAGS=$(CXXFLAGS) cargo build --package play-cpp-sdk --release
-  endif		
+	MACOSX_DEPLOYMENT_TARGET=10.15 CXX=$(CXX) CXXFLAGS=$(CXXFLAGS) cargo build --package play-cpp-sdk --release --target aarch64-apple-darwin
+	MACOSX_DEPLOYMENT_TARGET=10.15 CXX=$(CXX) CXXFLAGS=$(CXXFLAGS) cargo build --package play-cpp-sdk --release		
```

Comment:
If we are going to support arm64,  10.15 is outdated.

Reply:
this is for supporting ue4 , legacy x86_64 mac